config/log.go

```diff
@@ -41,6 +43,7 @@ func (mgr *LogManager) OnConfigChange(_ context.Context, cfg *Config) {
 	}
 
 	if cfg.Options.LogLevel != "" {
-		log.SetLevel(cfg.Options.LogLevel)
+		log.SetLevel(cfg.Options.LogLevel.ToZerolog().String())
+		zerolog.SetGlobalLevel(cfg.Options.LogLevel.ToZerolog())
```

Comment:
I think this is the only place that calls internal/log `SetLevel()`? Should we go ahead and update `SetLevel()` to take a `zerolog.Level` directly, and then we can delete the mapping from string back to level there? https://github.com/pomerium/pomerium/blob/a1388592d8627368ef9421c98d43d500e20beb37/internal/log/log.go#L69-L84

(Please feel free to disagree, I'd value your opinion on this.)